orttraining/orttraining/python/orttraining_pybind_state.cc

```diff
@@ -0,0 +1,239 @@
+#include "python/onnxruntime_pybind_exceptions.h"
+#include "python/onnxruntime_pybind_state_common.h"
+
+#include "core/framework/random_seed.h"
+#include "core/framework/session_options.h"
+#include "orttraining/core/session/training_session.h"
+#include "orttraining/core/graph/optimizer_config.h"
+#include "orttraining/core/framework/mpi_setup.h"
```

Comment:
nit: can the includes be in alphabetical order? also <pybind11/stl.h> should go first. see [https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)

Reply:
Good reading. If I understand, I put in order python (related),  std (standard), ort (other) headers.